packages/theme/src/colorsV2.ts

```diff
@@ -69,6 +69,7 @@ export const colorsV2 = {
         baseContentNeutral: paletteV2.lightCoolGrey600,
```

Comment:
why there are so many changes in this file? Didn't we use correct colors before?